packages/paypal-commerce-integration/src/PaypalCommerceRatePayPaymentMethod.tsx

```diff
@@ -79,6 +82,7 @@ const PaypalCommerceRatePayPaymentMethod: FunctionComponent<any> = ({
                 paypalcommerceratepay: {
                     container: '#checkout-payment-continue',
                     legalTextContainer: 'legal-text-container',
+                    onPending: (isLoading: boolean) => onPending && onPending(isLoading),
```

Comment:
```suggestion
                    onPending: (isLoading: boolean) => toggleLoadingOverlay(isLoading),
```

Reply:
"Call pending when pending"

It might be great to understand what this method do by reading the function name. From SDK perspective it has an option onPending that will be called (ideally) when the flow will get to this pending state. From UI perspective, you passing the function like: 'You should do that wen the callback will be called'.

So you can say -> 'Show me a loading overlay when the strategy will call onPending callback' -> or 'Hide the loading overlay if pending state finished'.

Reply:
Fixed